docs/troubleshooting.md

```diff
@@ -224,9 +224,10 @@ kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/releas
 
 The [CNI image](../scripts/dockerfiles/Dockerfile.release) built for the `aws-node` manifest uses Amazon Linux 2 as the base image. Support for other Linux distributions (custom AMIs) is best-effort. Known issues with other Linux distributions are captured here:
 
-- **iptables** - iptables is installed by default in `aws-node` container images. Newer distributions of RHEL (RHEL 8.x+), Ubuntu (Ubuntu 20.x+), etc. have moved to using `nftables`. This leads to issues such as [this](https://github.com/aws/amazon-vpc-cni-k8s/issues/1847) when running IPAMD.
+- **iptables**
+  Prior to v1.13.0, the VPC CNI image only contained `iptables-legacy`. Newer distributions of RHEL (RHEL 8.x+), Ubuntu (Ubuntu 21.x+), etc. have moved to using `nftables`. This leads to issues such as [this](https://github.com/aws/amazon-vpc-cni-k8s/issues/1847) when running IPAMD.
   
-  To resolve this issue on distributions that use `nftables`, there are currently two options:
+  To resolve this issue in versions before v1.13.0, there are currently two options:
```

Comment:
Why v1.13.0? Can't this be a patch version (i.e.,) v1.12.1?

Reply:
Sorry, yeah this should be `v1.12.1`. I forgot to update after concluding that #2149 will not make it in the next release. There are two other places where `v1.13.0` is referenced, so after this merges, I will put up a small MUT to update them.